src/Bicep.Core.UnitTests/Diagnostics/LinterRuleTests/UseResourceSymbolReferenceRuleTests.cs

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
```

Comment:
Is it important to support the following format where reference just takes the resource name?

reference(loadBalancerPublicIPAddressName).ipAddress

Reply:
Good question. This is hard to catch programmatically because there's potential ambiguity - it's possible to have multiple resources with the same name, and use of parameters may lead to inaccuracies. I didn't want to tackle this problem right now, but I'm definitely open to expanding this linter in the future if we feel like this is an important scenario to handle.

Reply:
k